hyo2/soundspeedsettings/widgets/output.py

```diff
@@ -234,7 +234,7 @@ def new_client(self):
         # port
         while True:
             # noinspection PyCallByClass
-            port, ok = QtWidgets.QInputDialog.getInteger(self, "New client", "Input the port (e.g., 4001)",
+            port, ok = QtWidgets.QInputDialog.getInt(self, "New client", "Input the port (e.g., 4001)",
                                                          4001, 0, 65535)
```

Comment:
The `4001, 0, 65535` should have been re-aligned to the content of the modified previous row. 

PyCharm provides a warning about this specific PEP8 issue:

![pycharm64_2019-01-26_10-33-46](https://user-images.githubusercontent.com/2849257/51789349-f5209200-2155-11e9-9074-8444bad7e146.png)

Do you mind to update this PR (no need to create a new PR) with this small issue fixed?